.github/workflows/build.yaml

```diff
@@ -127,9 +130,11 @@ jobs:
         name: Build on Darwin
         timeout-minutes: 60
 
+        strategy:
+            matrix:
+                type: [clang, python_lib]
```

Comment:
could we do this as a sequential build? matrix builds are costly in CI because they download the docker image and run bootstrap, both of which are slow.

Reply:
Updated.